PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,19 @@
-Thank you for submitting a pull request to the WrongSecrets app!
+Thank you for submitting a pull request to the WrongSecrets app! Please link any relevant issues.
 
-What kind of changes does this PR include?
+### What kind of changes does this PR include?
 
 -   [ ] Fixes or refactors
 -   [ ] A new challenge
 -   [ ] Additional documentation
 -   [ ] Something else
```

Comment:
```suggestion
-   [ ] Something else: document below.
```

Reply:
Addressed differently, how about this?

Reply:
can we make relations/references 1 header together? it feels a bit bulky now :) ... but maybe that is me